lib/eChronicler/journal_entries.ex

```diff
@@ -8,10 +8,20 @@ defmodule EChronicler.JournalEntries do
     Repo.all(JournalEntry)
   end
 
+  def reverse_chron_list_journal_entries() do
+    Repo.all(JournalEntry)
+    |> Enum.sort(&(&1.inserted_at > &2.inserted_at))
+  end
```

Comment:
It might be worth just making this be your `list_journal_entries()` function.

Reply:
Yes! I implemented this.